### PR TITLE
Minor grammar improvement

### PR DESCRIPTION
**Pull request by jhwohlgemuth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,8 +203,7 @@ whether that's effectively blocked on GC integration.
 
 To clear this up: **wasm is quite capable of working with the DOM today**. You
 can employ strategies like those in [`wasm-bindgen`][wasm-bindgen] to operate on
-the DOM via calls back into JS. However, such calls do impose an overhead, so
-efficiency gains are most easily had if once can batch up DOM
+the DOM via calls back into JS. However, such calls impose an overhead cost. Efficiency gains are most easily achieved by batching DOM
```
Comment (mgattozzi) [inline README.md]: Do you mind putting everything from Efficiency on in your change onto a newline so it's consitent with the rest of the README? Once you do let's get this merged :D Thanks for fixing it

Comment (jhwohlgemuth) [reply README.md]: Sure thing.

Comment (jhwohlgemuth) [reply README.md]: @mgattozzi New line added

**Comment by mgattozzi** [comment]
Thanks again @jhwohlgemuth! :D

**Pull request merged**